tests/backup/backup_test.go

```diff
@@ -58,7 +58,7 @@ const (
 	defaultTimeout       = 5 * time.Minute
 	defaultRetryInterval = 5 * time.Second
 
-	appReadinessTimeout = 10 * time.Minute
+	appReadinessTimeout = 20 * time.Minute
```

Comment:
let's try with a different pgbench app. @vinayakshnd  to share the app name.

Reply:
`pgbench-io-portion` is the app name